alt-integration-adapter-java/src/main/java/org/veriblock/protoservice/VeriBlockSecurityProtoService.java

```diff
@@ -327,4 +320,48 @@ public static GeneralReply checkATVInternally(VeriBlockMessages.AltPublication p
 
         return VeriBlockServiceCommon.validationResultToProto(result);
     }
+
+    public static VeriBlockMessages.GeneralReply updateContext(VeriBlockMessages.UpdateContextRequest request) {
```

Comment:
protoservice package should not contain library code. It should redirect calls to the security service instance.